src/components/Dashboard/scenes/ManageMentorApplications/ManageMentorApplications.tsx

```diff
@@ -139,7 +139,13 @@ export const ManageMentorApplications: React.FC = () => {
           <tbody className="bg-white">
             {filteredMentorsByName.map((mentor) => (
               <tr key={mentor.uuid}>
-                <td className="px-6 py-2 whitespace-no-wrap border-b border-gray-200 w-1/4">
+                <td
+                  className="px-6 py-2 whitespace-no-wrap border-b border-gray-200 w-1/4"
+                  onClick={() => {
+                    window.open(`/dashboard/mentor-application/${mentor.uuid}`);
+                  }}
+                  style={{ cursor: 'pointer' }}
```

Comment:
You can use the Link component from the router dom to navigate. Also `cursor-pointer` tailwind class to make it a cursor. But wrapping it with the link would do it by default/

Reply:
Updated with a24c510